apps/prometheus/deployers/run_litmus_test.yml

```diff
@@ -26,7 +26,7 @@ spec:
 
           - name: PROVIDER_STORAGE_CLASS
             # Supported values: openebs-standard, local-storage
```

Comment:
Can you update the comment here. It is bit misleading. You can just mention that openebs-storageclass has to be used here.